.gitignore

```
@@ -17,6 +17,10 @@
 # emacs
 flycheck*
 
+## Visual code settings
+.vscode/launch.json
+.vscode/settings.json
```

```suggestion
.vscode/
```
We can just ignore everything under `.vscode` because there are other files sometimes.